AdaptiveCards/templating/index.md

```diff
@@ -170,7 +179,7 @@ The Templating SDKs make it possible to populate a template with real-data.
 
 > [!NOTE]
 >
-> During the initial preview we only have a limited set of SDKs available. When we release there will be templating libraries for every supported Adaptive Cards platform.
+> At this time templating is targeted primarily for "backend" platforms (.NET and NodeJS). Over time we will release templating SDKs for all remaining Adaptive Cards platform, like iOS, Android, UWP, etc.
```

Comment:
Should we try to explain what we really mean by "backend" since it means different things in different contexts or just try to avoid the term altogether and be explicit about the targeted scenarios ? What is my backend maybe somebody else's frontend or middle tier for that matter.

Reply:
FYI, the [Microsoft writing guides](https://styleguides.azurewebsites.net/StyleGuide/Read?id=2700) have a handy checklist of things to note for developer targeting content and in general very handy I'm finding with its very specific prescriptive format. Worth a read